ClassicImplementation/ios/Podfile.lock

```diff
@@ -676,10 +676,10 @@ SPEC CHECKSUMS:
   RNCClipboard: 41d8d918092ae8e676f18adada19104fa3e68495
   RNScreens: ea4cd3a853063cda19a4e3c28d2e52180c80f4eb
   SocketRocket: f32cd54efbe0f095c4d7594881e52619cfe80b17
-  SwiftProtobuf: b70d65f419fbfe61a2d58003456ca5da58e337d6
+  SwiftProtobuf: 69f02cd54fb03201c5e6bf8b76f687c5ef7541a3
   Yoga: 065f0b74dba4832d6e328238de46eb72c5de9556
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
 PODFILE CHECKSUM: 480c6e12b43259494addde607249351b2ccfc198
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.14.2
```

Comment:
@cedancp-cs  I would suggest also updating Gemfile and Gemfile.lock to this version, so we are consistent with cocoapods versions across machines